LICENSE.md

```diff
@@ -0,0 +1 @@
+# TODO
```

Comment:
Are these necessary? I think it's standard to use plain-text. We can always follow up with individuals to provide more detailed and nicely formatted information if needed.